### PR TITLE
Removing hardcoded endpoints in overview page and Publisher thumbnail upload error

### PR DESCRIPTION
**Pull request by malinthaprasan**
- Removed hardcoded endpoint "http://192.168.1.2:8282/SwaggerPetstore/1.0.0" from Store overview page and used first https/http hybrid GW endpoint as a temporary fix.
- Fix ES lint issues
- Fixes: https://github.com/wso2/product-apim/issues/5010

Need to properly add URLs per each GW as per https://github.com/wso2/product-apim/issues/5003

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 1]
```diff
@@ -472,8 +472,7 @@ class InfoBar extends React.Component {
      */
     componentDidMount() {
         const api = new Api();
-        const promised_api = api.getAPIById(this.api_uuid);
-        promised_api
+        api.getAPIById(this.api_uuid)
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: I think we could use the API UUID from the props directly here in `componentDidMount`, No need to assign it to `this`

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 2]
```diff
@@ -543,12 +538,35 @@ class InfoBar extends React.Component {
...
+     *
+     * @param {Api} api API object
+     * @returns {string}
+     */
+    getHttpsEP = (api) => {
+        const epUrls = api.endpointURLs.filter((v) => {
+            if (v.environmentType === 'hybrid') {
+                return v;
+            } else {
+                return null;
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: I think we don't need to return null here, We could simplify this as.

```javascript
api.endpointURLs.filter( url => environmentType === 'hybrid' )
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 3]
```diff
@@ -543,12 +538,35 @@ class InfoBar extends React.Component {
...
+            }
+        });
+
+        const selectedEnvUrls = epUrls[0].environmentURLs;
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: In this case, we could use 

const selectedEnvUrls = api.endpointURLs.find(url => environmentType === 'hybrid')

selectedEnvUrls.https || selectedEnvUrls.http

Comment (malinthaprasan) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Fixed!

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx (hunk 2 above)
Comment (malinthaprasan) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Fixed!

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx (hunk 1 above)
Comment (malinthaprasan) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Fixed!

**Pull request merged**